False 'no-member' on inner class of NamedTuple

The following produces a 'no-member' error.

```python
from typing import NamedTuple


class Ax(NamedTuple):
    class Bx:
        b = 0


print(Ax.Bx.b)

>>> pylint_no_member.py:9:6: E1101: Class 'Ax' has no 'Bx' member (no-member)

```

However, if Ax doesn't inherent from NamedTuple then it's fine.

```
Ubuntu 20.04.1 LTS
pylint 2.4.4
astroid 2.3.3
Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
[GCC 9.3.0]
```

--- a/CONTRIBUTORS.txt
+++ b/CONTRIBUTORS.txt
@@ -25,7 +25,7 @@ Current team:
   Added Python 3 check for accessing deprecated methods on the 'string' module,
   various patches.
 
-* Dmitry Pribysh: committer
+* Dmitri Prybysh: committer
 
   multiple-imports, not-iterable, not-a-mapping, various patches.
 

--- a/tests/functional/n/no/no_member_nested_namedtuple.py
+++ b/tests/functional/n/no/no_member_nested_namedtuple.py
@@ -1,0 +1,8 @@
+from typing import NamedTuple
+
+class Ax(NamedTuple):
+    class Bx:
+        b = 0
+
+
+print(Ax.Bx.b)

--- a/tests/functional/n/no/no_member_nested_namedtuple.rc
+++ b/tests/functional/n/no/no_member_nested_namedtuple.rc
@@ -1,0 +1,10 @@
+[testoptions]
+min_pyver=3.7
+
+[MESSAGES CONTROL]
+disable=fixme,
+    logging-too-many-args,
+    logging-fstring-interpolation,
+    missing-docstring,
+    no-else-return,
+    too-few-public-methods,

